easybuild/easyconfigs/g/GATK/GATK-3.3-0-Java-1.8.0_66.eb

```diff
@@ -0,0 +1,42 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 Cyprus Institute / CaSToRC, University of Luxembourg / LCSB
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis.georgatos@uni.lu>, Kenneth Hoste (UGent)
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+
+easyblock = 'Tarball'
+
+name = 'GATK'
+version = '3.3-0'
+
+homepage = 'http://www.broadinstitute.org/gatk/'
+description = """The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute 
+ to analyse next-generation resequencing data. The toolkit offers a wide variety of tools,
+ with a primary focus on variant discovery and genotyping as well as strong emphasis on 
+ data quality assurance. Its robust architecture, powerful processing engine and 
+ high-performance computing features make it capable of taking on projects of any size."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# download manually from http://www.broadinstitute.org/gatk/download
+sources = ['GenomeAnalysisTK-%(version)s-g37228af.tar.bz2']
```

Comment:
Ugh, that sucks...

Since the download must be done manually anyway, I would stick to the name without the commit ID in it, and thus require people to rename after downloading in case they're downloading old versions?